### ISSUE TITLE
Donation pages (user side )

### ISSUE THREAD

**Issue by codewarnab** [Innovateninjas/Paws-frontend#294]

**Comment by codewarnab** [comment]
this issue is assigned to @NilanchalaPanda best of luck

**Comment by NilanchalaPanda** [comment]
Thanks for the update @codewarnab 🙌🏻 

Also can you review my previous PR. Looking forward for your response.

**Comment by NilanchalaPanda** [comment]
Hey @codewarnab and @rishicds, will this page look good for `DONATION PAGE` of Paws. Please review and share your thoughts.

**Comment by rishicds** [comment]
2nd link looks nice what do you think @codewarnab ?

**Comment by codewarnab** [comment]
> Hey @codewarnab and @rishicds, will this page look good for `DONATION PAGE` of Paws. Please review and share your thoughts. LINK 1 : https://htmlbeans.com/html/cuidar/10-Donate.html LINK 2 : https://halpes-nuxt.vercel.app/causes-details#

yes 2nd one looks good but don't try to exactly like that keep the theme of the current app and then build top of it .
you can add relevant animal related images and for phone you can just them if they make the page clutter in the phone screen 
also you a line/section of total donation which will be static for now and will be dynamic after we fully integrate payment gateway .
Best of luck just try to make it best looking

**Comment by codewarnab** [comment]
@NilanchalaPanda  any update .

**Comment by drick-infinity** [comment]
Assign this issue to me As i learned Mern stack development i would create better ui for donation page

**Comment by drick-infinity** [comment]
under gssoc '24

**Comment by Vaibhavsg17** [comment]
@codewarnab issue is still pending please assign me this issue.

**Comment by codewarnab** [comment]
> under gssoc '24

okay but start work after #439 is merged

**Comment by codewarnab** [comment]
> @codewarnab issue is still pending please assign me this issue.

its FCFS
